src/renderers/shared/fiber/ReactFiberClassComponent.js

```diff
@@ -41,7 +41,7 @@ var invariant = require('invariant');
 const isArray = Array.isArray;
 
 module.exports = function(
-  scheduleUpdate : (fiber : Fiber, priorityLevel : PriorityLevel) => void,
+  scheduleUpdate : (fiber : Fiber, priorityLevel : PriorityLevel, callerName : string | null) => void,
```

Comment:
I'm not a fan of this pattern since they don't get stripped in prod. Can we reword it to be `setState, replaceState or forceUpdate` in the message instead of complicating the code?